client/src/Components/Cursor/Cursor.js

```diff
@@ -53,7 +56,11 @@ const Cursor = (props) => {
     if (qs('.cursorPos') !== null && qs('.cursorPos').innerHTML === '	') { // We are after a tab character
       cursorEl.style.left = (position.left + 34) + 'px';
```

Comment:
We should try to avoid scaling by a constant pixel value. However, since this is a temp fix, if it works it works.